### PR TITLE
Mobile App - Netflix Clone

### PR DESCRIPTION
**Pull request by himanshu240601**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes.

### Description
Hi, I want to contribute to this repo by adding a netflix clone app created using flutter. I put this app under mobile applications/intermediate

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#779

### CODE REVIEW

**Comment by Kushal997-das** [comment]
@himanshu240601 Please mention the issue number that you been solved .

**Comment by himanshu240601** [comment]
> @himanshu240601 Please mention the issue number that you been solved .

I've opened an issue, #779 
Kindly check and assign the same.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@himanshu240601 Hello , Your `readme.md` file images are not loading please check it .
Also please try to remove those files those are not really important to this project .

**Comment by himanshu240601** [comment]
> @himanshu240601 Hello , Your `readme.md` file images are not loading please check it . Also please try to remove those files those are not really important to this project .

Hey!, I've made the following changes - 
- Updated the readme.md file in netflix_clone directory
- Added the netflix clone project link in the readme.md in mobile application directory (the link is for the main branch)
- Removed unnecessary files from my project (netflix_clone)

File: Mobile Applications/Intermediate/netflix_clone/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### <p align="center">App Screenshots</p>
+
+<p align="center">
+<img src="screenshots/1.jpg" width=200/>
+<img src="screenshots/2.jpg" width=200/>
+<img src="screenshots/3.jpg" width=200/>
+<img src="screenshots/4.jpg" width=200/>
+</p>
+
```
Comment (Kushal997-das) [inline Mobile Applications/Intermediate/netflix_clone/README.md:13]: App screenshot are not working in `readme.md` file  [here](https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/README.md)

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Please fix the mentioned things .

File: Mobile Applications/Intermediate/netflix_clone/README.md (hunk 1 above)
Comment (himanshu240601) [reply Mobile Applications/Intermediate/netflix_clone/README.md:13]: I've checked the readme and it's working fine. All the screenshots are loading correctly.
Screenshot 2023-01-11 110851

Comment (Kushal997-das) [reply Mobile Applications/Intermediate/netflix_clone/README.md:13]: No , it's not [check this](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

Comment (himanshu240601) [reply Mobile Applications/Intermediate/netflix_clone/README.md:13]: > No , it's not [check this](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

I've checked multiple times, there's nothing wrong. I don't know why the screenshots are not working on your side.
Should I remove these from the readme?

Comment (Kushal997-das) [reply Mobile Applications/Intermediate/netflix_clone/README.md:13]: No do not remove try to fix it . 

This is the [URL](https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/README.md) for same

Comment (himanshu240601) [reply Mobile Applications/Intermediate/netflix_clone/README.md]: Hey! I've made some changes to the readme file. It should work now. Please check [here!](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

File: Mobile Applications/Intermediate/netflix_clone/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## <p align="center">Netflix Clone - Flutter</p>
+
+<p align="center">
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/1.jpg" alt="Image 1" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/2.jpg" alt="Image 2" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/3.jpg" alt="Image 3" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/4.jpg" alt="Image 4" width="200" height="400"/>
+</p>
```
Comment (Kushal997-das) [inline Mobile Applications/Intermediate/netflix_clone/README.md:8]: Image not loading

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@himanshu240601 Images are not loading .

File: Mobile Applications/Intermediate/netflix_clone/README.md (hunk 2 above)
Comment (himanshu240601) [reply Mobile Applications/Intermediate/netflix_clone/README.md]: > Image not loading

try to check again now!
and if still not working could you please share the URL of the images that you're receiving

**Comment by himanshu240601** [comment]
great 👍

**Pull request merged**